test/functional/apps/visualize/_custom_branding.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+export default function ({ getService, getPageObjects }) {
+  const globalNav = getService('globalNav');
+  const PageObjects = getPageObjects(['common', 'home']);
+
+  describe('OpenSearch Dashboards custom logo', function customLogo() {
```

Comment:
Do you think we should create a functional test to verify it's navigates correctly. For example, we can route to one page and verify that by clicking on the nav logo that it does re-route you to the home.

Reply:
There is already a functional test for testing that. Do we want to make a new one after we change the logo?